lib/generators.nix

```diff
@@ -205,12 +205,14 @@ rec {
        (This means fn is type Val -> String.) */
     allowPrettyValues ? false,
     /* If this option is true, the output is indented with newlines for attribute sets and lists */
-    multiline ? false
+    multiline ? false,
+    /* If multiline is enabled, prefix all new lines with this string */
+    prefix ? ""
```

Comment:
what do we need this for?

Reply:
See the PR description for an example of `prefix` usage. Though I'm now realizing that this won't work for multiline strings. I'll see if I can make multiline strings work nicely, probably with `builtins.replaceStrings [ "\n" ] ..` or so

Reply:
Hm, I see. It’s probably okay to have this as an option.